src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageEditContainer.tsx

```diff
@@ -210,6 +205,7 @@ const WorkPackageEditContainer: React.FC<WorkPackageEditContainerProps> = ({
             add={expectedActivitiesUtil.add}
             remove={expectedActivitiesUtil.remove}
             update={expectedActivitiesUtil.update}
+            disabledItems={ea.map((ea) => ea.isResolved)}
```

Comment:
if each object now has a isResolved property... couldn't you just check that in the EditableTextInputList component instead of separating it into its own list here?

Reply:
pretty much every usage of the EditableTextInputList takes in an array of strings for the items. these are then used directly to put in the value of each string to the list. if we take out a resolved property from each item, it might break the component for other usages of items that don't have resolved properties. this took me a while to think about but if you have a better idea for doing this lmk

Reply:
hmmm ok